liquibase-standard/src/main/java/liquibase/changelog/DatabaseChangeLog.java

```diff
@@ -973,7 +974,9 @@ public boolean include(String fileName,
                            OnUnknownFileFormat onUnknownFileFormat,
                            ModifyChangeSets modifyChangeSets)
             throws LiquibaseException {
-        if (".svn".equalsIgnoreCase(fileName) || "cvs".equalsIgnoreCase(fileName)) {
+        String filenameWithoutPath = Paths.get(fileName).getFileName().toString();
+        boolean matchesHiddenFilename = HIDDEN_FILENAME_PATTERN.matcher(filenameWithoutPath).matches();
+        if ("cvs".equalsIgnoreCase(filenameWithoutPath) || matchesHiddenFilename) {
```

Comment:
- ignore a hidden file or version control system file.
- decision-logic use should be filename without path.